docs/strategy-advanced.md

```diff
@@ -229,3 +229,99 @@ for val in self.buy_ema_short.range:
 # Append columns to existing dataframe
 merged_frame = pd.concat(frames, axis=1)
 ```
+
+## Adjust trade position
```

Comment:
I would actually move this whole section to `strategy-callbacks.md`.

Currently `configuration.md` links to the callback page - which forwards you to the advanced docs for "details" - which is quite odd - and kindof unnecessary.

It's for sure an advanced topic (maybe add a warning box about that - or that it's still in beta testing?) - but so are other topics covered by callbacks (like selecting a custom price, depending on the logic you implement).

Reply:
Moved the documentation all into callbacks page.